doc/PlaysTacticsSkillsSummary.md

```diff
@@ -0,0 +1,69 @@
+
+| Offensive Plays    | Description |
+| ------------------ | ----------- |
+| Adaptive Formation | Repeatedly finds the best pass, until it has a good shot on goal |
+| Basic 122          | One robot will always try to get the ball and shoot while the others defend |
+| Two Side Attack    | Places two static receiver robots in the middle of the field and decides whether to pass to them or shoot |
+
+## Tactics
+For higher level plays, only Coordinated Pass and Defense are really used.
+
+| Tactics          | Description | Use When ... |
+| ---------------- | ----------- | ------------ |
+| Coordinated Pass | Moves one robot to the ball and the other to the receive point. When they both are ready, they will pass | Use when you want to pass the ball from one robot to another |
+| Defense          | Basic 2 robot defense that stays on the goalie box. Clears the ball when it comes close. | Use in most cases unless you are writing your own defense |
+| Defense Old      | Same as Defense, but significantly harder to read code-wise | Don't use |
+| Forward Pass     | Basically the same as Coordinated Pass, but ball capture is slightly different | Don't use since the new ball capture needs more testing / tuning |
+| Line Up          | Lines up all robots in a specified line | Use when you need robots out of the way for testing purposes |
+| One Touch Pass   | Finds the best point in front of the opponent goal to pass to and one touch it in | Don't use since it hasn't been tested / tuned |
+| Our Placement    | Places the ball according to the auto-ref commands | Don't use unless trying to change ball placement code |
+| Penalty          | Takes the penalty and tries to shoot | Don't use unless trying to change Our Penalty code |
+| Tune PID         | Moves one robot back and forth to tune the motion control | Don't use unless you are trying to tune the motion control |
+
+
+## Positions
+These are in the tactic folder, but slightly higher level than normal tactics. They contain the AI for a single position (or group) on the field. 
+
+Defender and Goalie are really only used on some specific restarts. Submissive Defender and Submissive Goalie are used in Defense. Defensive Forward and Simple Zone Midfielder are used in adaptive formation.
+
+| Positions              | Description |
+| ---------------------- | ----------- |
+| Coach                  | The most important tactic we have. It yells strategic messages at the other robots, celebrates on goals, and follows the play from the side of the field. | 
+| Defender               | Defends specific areas of the field. Only used on restarts |
+| Goalie                 | Standalone goalie. If your play (or sub plays) does not contain code for the goalie, this one is used. |
```

Comment:
I think it would be nice to stress here that using goalie should be avoided whenever possible, since it dosen't coordinate with the defense at all. Maybe we should add some comments in the file as well (maybe there's an issue already open for this).

Reply:
I think it's #947